networkx/drawing/tests/test_layout.py

```diff
@@ -388,6 +396,14 @@ def test_spiral_layout_equidistant(self):
         dist = np.linalg.norm(p[1:] - p[:-1], axis=1)
         assert np.allclose(np.diff(dist), 0, atol=1e-3)
 
+    def test_forceatlas2_layout(self):
+        # check whether partial pos input still returns a full proper position
+        G = self.Gs
+        pos = nx.random_layout(G)
+        del pos[list(G.nodes())[0]]
```

Comment:
```suggestion
        del pos[next(iter(G))]
```
you could also use `nx.utils.arbitrary_element(G)`